dev/run-small-python-tests.sh

```diff
@@ -7,6 +7,6 @@ trap 'err=1' ERR
 export MLFLOW_HOME=$(pwd)
 
 # NB: Also add --ignore'd tests to run-large-python-tests.sh
-pytest --ignore-flavors
+pytest --ignore-flavors --requires-ssh
```

Comment:
<img width="940" alt="image" src="https://user-images.githubusercontent.com/17039389/167966999-87873326-728a-4c88-a4b2-ef2e9baf1aba.png">

Reply:
Confirmed pytest runs tests in `test_sftp_artifact_repository.py`.